### ISSUE TITLE
IDEA: Blockcart dropdown as pure Bootstrap dropdown component?

### ISSUE THREAD

**Issue by ghost** [PrestaShop/community-theme-16#269]
Hi!
I made yesterday almost successful adopt (experiment) of Bootstrap dropdown component as Blockcart dropdown but it stuck in ajax/json/js populating of new added products and remove them. Im searching for way to completly rework this old fashioned module. Anybody works on it? Thanks :)

The main reason:
- Wider usage Bootstrap 3 styles, utilities and components ;)
- Remove hovering and a lot of dependancies from ajax-cart.js
- Better mobile view and usage in top menu

**Comment by ghost** [comment]
And the another IDEA
Vertical and horizontal centered of #layer_cart 

In my way i prepend #layer_cart from 
`$('#columns #layer_cart, #columns .layer_cart_overlay').detach().prependTo('#columns');`
to
`$('#header #layer_cart, #header .layer_cart_overlay').detach().prependTo('#columns');
`
and change
`$('#layer_cart').css({'top': n}).fadeIn('fast');`
to
`$('#layer_cart').css({top:'50%',left:'50%',marginLeft:'-527px',marginTop:'-137px',position:'fixed',height:'275px',width:'1054px',zIndex:'9999',animation:'showSweetAlert 0.3s'}).fadeIn('fast');`

and its ideal and always on center even when window is scrolled. Now i need to find a way to better show on mobile - now #layer_cart is disabled on mobile.
